controllers/tests/controller/selfnoderemediation_controller_test.go

```diff
@@ -391,20 +388,6 @@ func deleteVolumeAttachment(vaName string, verifyExist bool) {
 	ExpectWithOffset(1, err).To(Succeed())
 }
 
-func verifyVaDeleted(vaName string) {
```

Comment:
Is there a reason not to verify VA deletion?

Reply:
VA are not deleted anymore, it's now up to KCM to handle the VAs

Reply:
Also in unit testing? It makes sense not to test that in e2e which resembles reality but in unit-test, there is no KCM, right?

Reply:
> in unit-test, there is no KCM, right?

That is correct, but because of the fact that there is no KCM the VA will never be deleted (SNR does not delete the VA anymore, and KCM will not delete them), so it make no sense to verify they are deleted - this sort of verification will always fail.

Reply:
Oh, now I get it.